config/config.go

```diff
@@ -634,6 +640,8 @@ func (c *Config) ToOptions() skipper.Options {
 			Prepend: c.PrependFilters.filters,
 			Append:  c.AppendFilters.filters,
 		},
+		Clone:              eskip.NewClone(c.CloneRoute.Reg, c.CloneRoute.Repl),
```

Comment:
I think this also should be called CloneRoute and EditRoute

Reply:
renamed

Reply:
I think fixed